### PR TITLE
feat: New Crowdin updates

### PR DESCRIPTION
**Pull request by teolemon**

### CODE REVIEW

File: po/common/common.pot [hunk 1]
```diff
@@ -16,6 +16,11 @@ msgctxt "1_product"
 msgid "1 product"
 msgstr ""
 
+# leave a space before and after, unless there are no spaces between "A and B" in the target language
+msgctxt "_and_"
+msgid " and "
+msgstr ""
+
```
Comment (teolemon) [inline po/common/common.pot:23]:
```suggestion
```

File: po/common/en.po [hunk 2]
```diff
@@ -19,6 +19,11 @@ msgctxt "1_product"
 msgid "1 product"
 msgstr "1 product"
 
+# leave a space before and after, unless there are no spaces between "A and B" in the target language
+msgctxt "_and_"
+msgid " and "
+msgstr " and "
+
```
Comment (teolemon) [inline po/common/en.po:26]:
```suggestion
```

File: po/common/ne.po [hunk 3]
```diff
@@ -2344,7 +2344,7 @@ msgstr ""
 
 msgctxt "site_name"
 msgid "Open Food Facts"
-msgstr "Open Food Facts"
+msgstr ""
```
Comment (teolemon) [inline po/common/ne.po:2347]:
```suggestion
msgstr "Open Food Facts"
```

File: po/common/pt_BR.po [hunk 4]
```diff
@@ -2360,7 +2360,7 @@ msgstr "Um banco de dados colaborativo, livre e aberto, com ingredientes, inform
 
 msgctxt "site_name"
 msgid "Open Food Facts"
-msgstr "Open Food Facts"
+msgstr ""
```
Comment (teolemon) [inline po/common/pt_BR.po:2363]:
```suggestion
msgstr "Open Food Facts"
```

File: po/openbeautyfacts/bo.po [hunk 5]
```diff
@@ -126,7 +126,7 @@ msgstr ""
 # Do not translate
 msgctxt "footer_obf"
 msgid "Open Food Facts"
-msgstr "Open Food Facts"
+msgstr ""
```
Comment (teolemon) [inline po/openbeautyfacts/bo.po:129]:
```suggestion
msgstr "Open Food Facts"
```

File: po/openbeautyfacts/sr.po [hunk 6]
```diff
@@ -126,7 +126,7 @@ msgstr ""
 # Do not translate
 msgctxt "footer_obf"
 msgid "Open Food Facts"
-msgstr "Open Food Facts"
+msgstr ""
```
Comment (teolemon) [inline po/openbeautyfacts/sr.po:129]:
```suggestion
msgstr "Open Food Facts"
```

File: html/donate/en.html [hunk 7]
```diff
@@ -247,7 +247,7 @@ <h3>Where your donation goes</h3>
             <img src="https://static.openfoodfacts.org/images/misc/2023-donation/join-the-food-rev.png"
                  alt="Join the food revolution!" style="margin-top: -15%; margin-bottom: -20%; width:80%"
             >
-            <!-- <h3>Where your donation goes</h3>
```
Comment (stephanegigandet) [inline html/donate/en.html:250]: it won't be seen, but the change seems broken

**Review by stephanegigandet — APPROVED** [review]

**Pull request merged**